modules/indexExchangeBidAdapter.js

```diff
@@ -419,6 +426,9 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     _IndexRequestData.targetAggregate = {'open': {}, 'private': {}};
 
     if (!utils.hasValidBidRequest(bidArr[0].params, requiredParams, ADAPTER_NAME)) {
+      for (var i = 0; i < bidArr.length; i++) {
+        passOnBid(bidArr[i].placementCode);
+      }
```

Comment:
Can you explain this logic to me of passing on all bids if the first bid is invalid?